lib/fizzy/parser_expr.cpp

```diff
@@ -63,6 +63,36 @@ struct ControlFrame
     {}
 };
 
+enum class OperandStackType : uint8_t
+{
+    Unknown = 0,
+    i32 = static_cast<uint8_t>(ValType::i32),
```

Comment:
`enum class ValType : uint8_t` so why is there a need for typecasting here?

Reply:
Couldn't this just extend `ValType`?

Reply:
Because `ValType` is still a different type from `uint8_t`, enum classes are strongly typed unlike plain enums.

Reply:
> Couldn't this just extend `ValType`?

No, not in C++